src/lib/journey/callbacks/choice/choice.svelte

```diff
@@ -17,6 +19,9 @@
   const choiceOptions = callback
     .getChoices()
     ?.map((label, idx) => ({ text: label, value: `${idx}` }));
+  const choiceOptionsForRadio = callback
+    .getChoices()
+    ?.map((label, idx) => ({ text: label, value: idx }));
```

Comment:
I had to duplicate `choiceOptions` here since the Radio composition expects the choices to be formatted as `{ text: string, value: number } `as opposed to `{ text: string, value: string }` which is what Select expects. I would welcome suggestions here for how we can improve this.

Reply:
I see this as a type bug. Let's fix it, so we don't have to duplicate the data structure. Honestly, the actual DOM treats the value as a string, regardless of what you pass in, so typing it as a number is a bit misleading. This change will have to be done for the primitive and the two compositional components (animated and standard).

Reply:
This was easy enough to fix I think - @cerebrl please take another look. Thanks.

Reply:
This looks great!